README.md

```diff
@@ -13,7 +13,7 @@
 
 Rye is a comprehensive project and package management solution for Python.
 Born from [its creator's](https://github.com/mitsuhiko) desire to establish a
-one-stop-shop for all Python users, Rye provides a unified experience to install and manages Python
+one-stop-shop for all Python users, Rye provides a unified experience to install and manage Python
```

Comment:
Not sure if the intent was instead something like "... Rye provides a unified experience for managing ..."

I read "to install" as "...to install [and manage] Python distributions"